utilities/kueue_utils.py

```
@@ -0,0 +1,142 @@
+    """
+    Context manager to create and optionally delete a ResourceFlavor.
+    """
+    try:
+        with ResourceFlavor(
+            client=client,
+            name=name,
+            teardown=teardown,
+        ) as resource_flavor:
+            yield resource_flavor
+    finally:
+        resource_flavor.remove_fin
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)